addons/l10n_mx_pos/models/pos_order.py

```diff
@@ -0,0 +1,65 @@
+from odoo import models, _
+class PosOrder(models.Model):
+    _inherit = 'pos.order'
+
+    def action_invoice_order(self):
+        self.env.context.update({
+            'state' : 'draft'
+        })
+        return {
+            'type': 'ir.actions.act_window',
+            'name': 'Select a Customer',
+            'view_type': 'form',
+            'view_mode': 'form',
+            'view_id' :  self.env.ref('l10n_mx_pos.global_customer_select_form').id,
+            'res_model' : 'pos.order.select',
+            'target' : 'new',
+            'flags': {'form': {'action_buttons': False}},
+            'context' : self.env.context
+        }
+
+    def _generate_multi_pos_order_invoice(self):
+
+        configs = {}
+        moves = self.env['account.move']
+
```

Comment:
Several checks missing here:
- what happen in case you select order that have already been "invoiced"
- what happen in case you have order with partner already set
- what if no line is selected
For case 1 and 2 you can decide to either filter them out or throw a warning
For case 3 I would throw a warning asking to select at least a pos order